default-recommendations.rkt

```diff
@@ -24,6 +25,7 @@
          resyntax/default-recommendations/legacy-contract-migrations
          resyntax/default-recommendations/legacy-struct-migrations
          resyntax/default-recommendations/let-binding-suggestions
+         resyntax/default-recommendations/conditional-suggestions
```

Comment:
The rules also need to be added to the `#:rules` list in the definition of `default-recommendations` below.